app/src/components/Announcement.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <div class="announcements">
+        <p class="mt-4 common ann-text fade-in" v-if="announcement != null">
+            <i class="fas fa-scroll-old"></i> {{announcement['msg']}}
+        </p>
+    </div>
+    
+</template>
+
+<script>
+    export default {
+        name: "Announcement",
+
+        data() {
+            return {
+                announcement: null
+            }
+        },
+
+        created() {
+            this.getAnnouncement();
+        },
+
+        methods: {
+            getAnnouncement: function () {
+                setInterval(() => {
+                    fetch(this.$root.$data['api']+":27522/api/announcement")
```

Comment:
The port should be configurable too, you never know when someone might change it...

Reply:
Done.